varlink/connection.go

```diff
@@ -84,6 +84,13 @@ type ReadWriterContext interface {
 	ReadBytes(ctx context.Context, delim byte) ([]byte, error)
 }
 
+// GetNetConn allows access to the underlying net.Conn (where one exists)
+// You shouldn't use this for I/O - but might use it to do things like access
+// peer credentials on a unix socket
+type GetNetConn interface {
```

Comment:
The method name was picked to match the equivalent [tls.Conn](https://pkg.go.dev/crypto/tls#Conn.NetConn) method